src/vrm/humanoid/VRMHumanoid.ts

```diff
@@ -55,9 +59,33 @@ export class VRMHumanoid {
         return;
       }
 
+      // Take a diff from restPose
+      // note that restPose also will use getPose to initialize itself
+      _v3A.set(0, 0, 0);
+      _quatA.identity();
+
+      const restState = this.restPose[vrmBoneName];
+      if (restState?.position) {
+        _v3A.fromArray(restState.position).negate();
+      }
+      if (restState?.rotation) {
+        _quatA.fromArray(restState.rotation).inverse();
+      }
+
+      // 🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥
+      if (!restState) {
+        if (vrmBoneName === VRMSchema.HumanoidBoneName.LeftUpperArm) {
+          node.rotation.x = 0.5 * Math.PI;
```

Comment:
Should this be something like `PI_HALF` constant? like
https://github.com/pixiv/three-vrm/blob/f10623dbcf7470624dafbc6bb10302f8267d2308/src/vrm/material/MToonMaterial.ts#L8

Reply:
oh no

it's just a leftover code. I completely forgot to remove despite I've put a lot of 🔥 on it. Let me just remove these lines 🙇 🙇 🙇